src/components/AppStart/index.js

```diff
@@ -0,0 +1,4 @@
+// @flow
+export { triggerLoadCoreEpic, loadAppEpic, loadCoreFailedEpic } from './appStart.epics';
+export { actionTypes as appStartActionTypes } from './appStart.actions';
```

Comment:
In the same spirit as in the other comment. I believe avoiding using the `as` would be helping us in the long term with searches in the codebase and stuff. 

Open for discussion if there are any advantages of using it here but my I would name the variable directly `appStartActionTypes`.